index.html

```diff
@@ -1,153 +1,183 @@
-        </div>
-        <div class="work">
-          <h3>Profesional Art Printing Data</h3>
-          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-          <ul class="languages">
-            <li class="lang">html</li>
-            <li class="lang">bootstrap</li>
-            <li class="lang">ruby</li>
-          </ul>
-          <button class="seeWork" type="button" name="seeWork">See Project</button>
-        </div>
-      </section>
+        <button id="resume" type="button" name="resume">Get My Resume</button>
+      </div>
+      <!-- ↓PROFICIENCY↓ -->
+      <div id="proficiency">
+        <ul class="prof">
+          <li id="profLanguage">
+            <h3>Language</h3>
+          </li>
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>Html</li>
+          <li>Css</li>
+        </ul>
+        <ul class="prof">
+          <li id="profFrame">
+            <h3>Frameworks</h3>
+          </li>
+          <li>Bootstrap</li>
+          <li>Ruby on Rails</li>
+          <li>RSPec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+        <ul class="prof">
+          <li id="profSkills">
+            <h3>Skills</h3>
+          </li>
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </div>
+    </section>
+  </main>
 
-      <section id="about">
-        <!-- ↓ABOUT↓ -->
-        <div id="me">
-          <h2 id="meTitle">About me</h2>
-          <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-          <button id="resume" type="button" name="resume">Get My Resume</button>
-        </div>
-        <!-- ↓PROFICIENCY↓ -->
-        <div id="proficiency">
-          <ul class="prof">
-            <li id="profLanguage"><h3>Language</h3></li>
-            <li>Javascript</li>
-            <li>Ruby</li>
-            <li>Html</li>
-            <li>Css</li>
-          </ul>
-          <ul class="prof">
-            <li id="profFrame"><h3>Frameworks</h3></li>
-            <li>Bootstrap</li>
-            <li>Ruby on Rails</li>
-            <li>RSPec</li>
-            <li>CapyBara</li>
-            <li>Selenium</li>
-          </ul>
-          <ul class="prof">
-            <li id="profSkills"><h3>Skills</h3></li>
-            <li>Codekit</li>
-            <li>Github</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Terminal</li>
-          </ul>
-        </div>
+  <footer>
+    <!-- ↓CONCTACT↓ -->
+    <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+    <form id="form" action="index.html" method="post">
```

Comment:
- [ ] Please collect data by using the [Formspree service](https://formspree.io/html). Action value should be similar to `https://formspree.io/f/{form_id} `:+1: